packages/hub-threads-client/package.json

```diff
@@ -12,21 +12,11 @@
     "access": "public"
   },
   "scripts": {
-    "prepublishOnly": "npm run build",
```

Comment:
you'll notice these are all different now. we _don't_ automatically try to rebuild on install. build is now an explicit step, that is _required_ before a publish/release. If you use the top-level release script, it'll do it automatically, so do that. But this also means you'll have to "manually" call npm run build if you make significant changes during build time. the good news is you can use watch, and the build will be much faster.